PyInstaller/hooks/hook-faker.py

```diff
@@ -0,0 +1,14 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2005-2019, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+from PyInstaller.utils.hooks import collect_submodules, collect_data_files
+
+hiddenimports = collect_submodules('faker.providers')
+datas = (collect_data_files('text_unidecode') +
```

Comment:
Travis tests fail on this line because they want the plus (+) moved down to the beginning of the next line.

Reply:
@cclauss what is the reason for the failure this time?

Reply:
See that 🚫 on the last line below...  That means that file does not end with an empty line... let’s hope that flake8 does not complain about that...